haiku-misc/neonlights/neonlights-0.0.1.recipe

```diff
@@ -0,0 +1,41 @@
+SUMMARY="Screensaver drawing neon lines"
+DESCRIPTION="NeonLights is a screensaver that draws colorful neon lines. \
+It was based loosely on Jared Tarbell's work. Inspired by Atalax's Substrate."
+HOMEPAGE="https://github.com/pulkomandy/neonlights/"
```

Comment:
No need for the trailing slash in HOMEPAGE. You can use:
```
HOMEPAGE="https://github.com/pulkomandy/neonlights"
```
Of course, this depends on the web site (or URI). Some sites do need the trailing slash for certain URIs.
Remark: if the URI had no path part (e.g. https://google.com/, then keeping the trailing slash would have been required in order to have a valid URI, although some browsers chose to hide these trailing slashes when there is no path part.